cinder/manifests/init.pp

```diff
@@ -261,6 +261,9 @@
   $default_availability_zone   = false,
   $enable_v1_api               = true,
   $enable_v2_api               = true,
+  $auth_uri                    = 'http://localhost:5000/',
+  $identity_uri                = 'http://localhost:35357/',
+  $nova_pub_url                = 'http://localhost:8776/',
   # DEPRECATED PARAMETERS
```

Comment:
move them to api.pp and overwrite the default values by specifying in hiera.yaml file.